### DISCUSSION TITLE
Demo Script and milestones

### DISCUSSION THREAD

**Discussion by AnkitaK26** [thinkarlabs/P23-004-MalnutritionAwareness#37] [General]
1. Onboarding screens
    a. Splash Screen
    b. The significance of the first 1000 days—awareness material 
    c. Nutrition: Information on nutrition prior to, during, and after pregnancy as well as postpartum nutrition 
    d. Important to measure child every month- Explain to the parent and any other carers why it's crucial that they frequently measure 
        their child.
2. Beneficiary stage selection (Pregnant woman, lactating mother, 6-24 months)
3. Account registrations for Pregnant woman, lactating mother and 6-24 months.
4. OTP verification flow
5. Login flow
6. Home Screen - Milestones
7. Track Health - user can track the health status of his or her child (MAM / SAM)
8. Ask - chat on WhatsApp
9. User Profile
10. Language - translation to selected language

**Milestone 1: 1, 2, 3, 4, 5, 6, 7, 8 - 23rd March
Milestone 2: 9, 10**

**Comment by srivatsapraveen** [comment]
The above demo script only on-boards the beneficiary - right?
What all will we include as features for the MVP (minimum viable product) release? It will be great if we can put this and break this down into one or more milestones (with into on when we are targeting these milestones)
